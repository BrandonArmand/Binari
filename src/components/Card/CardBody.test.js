import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import Cardbody from "./Cardbody";

describe("Cardbody", () => {
    it("renders snapshot", () => {
        const { container } = render(<Cardbody />);
        expect(container).toMatchSnapshot();
    });
    it("renders Cardbody", () => {
        render(<Cardbody>Cardbody</Cardbody>);
        expect(screen.getByText("Cardbody")).toBeInTheDocument();
    });
});
