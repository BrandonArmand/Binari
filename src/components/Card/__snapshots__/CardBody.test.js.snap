// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cardbody renders snapshot 1`] = `
<div>
  <div
    class="makeStyles-cardBody-1"
  />
</div>
`;
